nature-logger/Cells/CustomTableViewCell.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  CustomTableViewCell.swift
+//  nature-logger
+//
+//  Created by Frederik Helth on 15/12/2020.
+//  Copyright © 2020 Syddansk Universitet. All rights reserved.
+//
+
+import UIKit
+
+class CustomTableViewCell : UITableViewCell {
+    
+    @IBOutlet weak var plantTitle: UILabel!
+    @IBOutlet weak var plantCell: UIView!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        // Initialization code
+    }
```

Comment:
does this function do something? It seems to be only propagating the awakeFromNib